streamer/src/quic.rs

```diff
@@ -98,8 +97,7 @@ pub(crate) fn configure_server(
 }
 
 fn rt() -> Runtime {
-    Builder::new_multi_thread()
-        .worker_threads(NUM_QUIC_STREAMER_WORKER_THREADS)
+    tokio::runtime::Builder::new_multi_thread()
```

Comment:
am I right that it will mean that default number of threads will be used which is number of cores? Or we redefine somewhere `TOKIO_WORKER_THREADS`? 
I wonder what is the reason for this change and if it is directly correlated with other changes here.